src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -51,11 +63,42 @@ class CloseOrderModal extends Component {
         >
           <FormattedMessage id="ui-orders.paneBlock.closeBtn" />
         </Button>
-        <CloseOrderModalForm
-          close={this.closeModal}
-          orderId={orderId}
+        <Modal
+          label={<FormattedMessage id="ui-orders.closeOrderModal.title" values={{ orderId }} />}
           open={this.state.openModal}
-        />
+        >
+          <Row>
+            <Col xs={12}>
+              <Select
+                placeHolder="---Choose the reason---"
+                dataOptions={reasons}
+                required
+                label={<FormattedMessage id="ui-orders.closeOrderModal.reason" />}
+                onChange={(e => this.setState({ values: { reason: e.target.value } }))}
+              />
+              <TextArea
+                label={<FormattedMessage id="ui-orders.closeOrderModal.notes" />}
+                onChange={(e => this.setState({ values: { note: e.target.value } }))}
+              />
+            </Col>
+          </Row>
+          <Button
+            buttonStyle="primary"
+            onClick={() => {
+              closeOrderSubmit(this.state.values);
+              this.closeModal();
+            }}
+            style={{ float: 'right' }}
+          >
+            <FormattedMessage id="ui-orders.closeOrderModal.submit" />
+          </Button>
+          <Button
+            onClick={() => this.closeModal()}
+            style={{ float: 'right', marginRight: '10px' }}
```

Comment:
again, are you sure that layout by floats is a common way in Folio? I don't think so...